packages/strapi-hook-mongoose/lib/service/crud.js

```diff
@@ -0,0 +1,48 @@
+const { has, isEmpty, unionWith, isEqual, get } = require('lodash');
+const { buildQueryJoins, buildQueryFilter } = require('./utils');
+const { WhereBuilder, FilterBuilder } = require('../utils/query');
+
+class CRUDService {
```

Comment:
For now this service handles only two services (FetchAll and count), It'll be used to extend all the API Services